spec.emu

```diff
@@ -33,9 +33,9 @@ contributors: "Jack Works"
         <h1><dfn>CreateNumericRangeIterator</dfn>(_start_, _end_, _option_, _type_)</h1>
         <emu-alg>
             1. If Type(_start_) is not _type_, throw a *TypeError* exception.
```

Comment:
Actually this step uses `_type_` as a runtime ECMAScript value for comparison of `typeof _start_ === _type_`. Let me fix this.

Reply:
The `Type` abstract operation is not the same thing as the ECMAScript `typeof` operator. In particular, it returns the name of a type, not an ES language string (or a spec enum).